src/TrimDeps/Program.cs

```diff
@@ -0,0 +1,58 @@
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        ChangeEntryPointLibraryName(args[0], null);
```

Comment:
Sorry, should have added WIP to commit message, I copied this from the code I wrote to build shared runtime I'm going to trim it today.